Automattic/simplenote-macos
# Sharing is Caring

_Depends on #135_

Take advantage of the macOS `NSSharingService` (available as of 10.8) to allow the content of a note to be shared out to whatever services are on the system.

I also removed the publishing option from the info menu and added it as an option in the share menu.

<img width="317" alt="screen shot 2017-11-13 at 2 39 47 pm" src="https://user-images.githubusercontent.com/789137/32753565-32c996b6-c882-11e7-8270-1d1c8b905b2e.png">

**To Test**
* Fire up the app, select a note and click on the new publish option.
* Select from a service of your choice, did it share the note content properly?
* Also test the 'Publish to Web' option to ensure it works properly.
* Create a new note, the share button should be disabled until the note has some content.
